code/common/datadog.q

```diff
@@ -0,0 +1,36 @@
+// DATADOG CHECKS
+
+//Create datadog namespace
+\d .dg
+
+ddconfigfile:@[value;`ddconfigfile;hsym `$getenv[`KDBAPPCONFIG],"/ddconfig.txt"]
+
+//sets dogstatsd_port to the port defined by ddconfigfile
+$[`ddconfig.txt in key hsym `$getenv[`KDBAPPCONFIG];value each read0 ddconfigfile;dogstatsd_port:8125]
+
+//Functions are set to return 1b or 0b based on the health of the service.
+//Default check returns 1b from each process to indicate process is up and can be queried.
+//These checks run every time crontab/timer is run.
+
+handlers:(`symbol$())!()
+
+is_ok:{[x]
+        f:$[@[{[x].proc.proctype in key .dg.handlers};`;0b];
+                .dg.handlers[.proc.proctype];
+                .dg.default_is_ok];
+        @[f;`;0b]
+        }
+
+default_is_ok:{[x]1b}
```

Comment:
If there is no intention of having this overwritten then it should just be a lambda in `is_ok`.